package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "webpacktutorial",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "serve": "webpack serve"
```

Comment:
- Please, let's add the "start" to make our server start properly as the sever is not starting now.
   ` "start": "webpack serve --open"`